tests/integration/testdata/start_api/methods-resources-api-template.yaml

```diff
@@ -0,0 +1,127 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I assume once the intrinsics PR gets merged and integrated, you will add some testdata and integ tests for resource and methods that are defined with intrinsics?

Reply:
Ye. A lot of the code resource and methods is based on intrinsic references. When that is merged, all the tests will be updated to use intrinsics.